src/config.ts

```diff
@@ -1,3 +1,19 @@
 export const API_URI = process.env.API_URI || "http://localhost:8000/graphql/";
 export const TEST_AUTH_EMAIL = process.env.TEST_AUTH_EMAIL || "";
 export const TEST_AUTH_PASSWORD = process.env.TEST_AUTH_PASSWORD || "";
```

Comment:
We don't need envs for those either, since we're not using polly

Reply:
We need, we still use polly in `user.test.ts`